db/migrations/mysql/000023_create_threads.up.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS Threads (
+    PostId varchar(26) NOT NULL,
+    ChannelId varchar(26) DEFAULT NULL,
+    ReplyCount bigint(20),
+    LastReplyAt bigint(20),
+    Participants text,
+    PRIMARY KEY (PostId)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+SET @preparedStatement = (SELECT IF(
+    (
+        SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'Reactions'
```

Comment:
As this alter is affecting Reactions, is this its PR? We're missing an alter for ChannelId, so this might be the original paste without modifications for that one...